playbooks/roles/openvpn/tasks/main.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: Import the public GPG key from OpenVPN community
+  apt_key: url=https://swupdate.openvpn.net/repos/repo-public.gpg state=present
```

Comment:
Instead of doing it like this and pulling the key from a remote location, it's a little more secure to pin the GPG key in the repo. You can see an example [here](https://github.com/jlund/streisand/blob/master/playbooks/roles/nginx/tasks/main.yml#L7).